### ISSUE TITLE
Can Gzip unarchive return Member?

### ISSUE THREAD

**Issue by HouHangLei** [tsolomko/SWCompression#33]
After unzipping I need to get the file name (mainly the file suffix) so it can be stored in the sandbox directory.

**Comment by tsolomko** [comment]
Yes, you can use `GzipArchive.multiUnarchive(archive:)` instead. It works even on a single GZip archive, in which case it returns an array with only a single `Member`.
